packages/shared/src/dto/workflows/workflow-commons-fields.ts

```diff
@@ -80,4 +80,5 @@ export type PreferencesResponseDto = {
 
 export type PreferencesRequestDto = {
   user: WorkflowPreferences | null;
+  workflow?: WorkflowPreferences | null;
```

Comment:
In case of promotion, we want to clone also workflow preferences from dev -> prod.